cypress/e2e/Test Cases/Test Case-4.cy.js

```diff
@@ -0,0 +1,17 @@
+// cypress/integration/lockedOutUser.spec.js
+
+describe('Locked-Out User Test', () => {
+    it('Locked-out user cannot log in with valid credentials', () => {
+      // Visit the login page
```

Comment:
All comments which are obvious are unnecessary. 

example
checkoutPage.sortItemsListByOrder();
Below there an obvious comment which never should get into the code.
//Checkout Page - sort Items by Order